sdks/java/io/elasticsearch-tests/elasticsearch-tests-7/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -46,63 +45,39 @@
 
 /** Tests for {@link ElasticsearchIO} version 7. */
 @ThreadLeakScope(ThreadLeakScope.Scope.NONE)
-// use cluster of 1 node that has data + master roles
-@ESIntegTestCase.ClusterScope(scope = SUITE, numDataNodes = 1, supportsDedicatedMasters = false)
-public class ElasticsearchIOTest extends ESIntegTestCase implements Serializable {
+public class ElasticsearchIOTest implements Serializable {
```

Comment:
Is it time to think about merging classes to avoid all the duplication perhaps?

I know it's not created within this PR, but the similarities in the ES6 and ES7 tests look like they should be candidates to be a single class that takes a container tag to run against. I didn't check the ES5 one and I'm afraid I don't know the background as to why we have separate classes.

We might consider taking that as a separate Jira.

Reply:
@timrobertson100 +1 I agree, I'd love to see the test files consolidated if that were possible. Also +1 on that being a separate jira 😝